docs/DeveloperGuide.md

```diff
@@ -741,20 +741,107 @@ testers are expected to do more *exploratory* testing.
 
 ### Deleting a person
 
-1. Deleting a person while all persons are being shown
+#### Deleting a person successfully
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+**Steps:**
+1. Test case: `delete 5`
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+**Expected Result:**
+1. The contact information of the person at the specified index of the list is deleted from FINDvisor.
+2. *Person List* updates the list of contact without the deleted person.
+3. *Command Result Box* outputs a successful execution message with the deleted person's information.
+4. *Command Box* is cleared.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+**Alternative Test Cases:**
+- `delete 3`
+- `delete 1`
 
-1. _{ more test cases …​ }_
+Alternative test cases should have similar expected results.
```

Comment:
LGTM but I would like to suggest putting this at the top of the entire section as a note since all of our test cases will have this line.